tests/test_rhiza/test_makefile.py

```
@@ -244,3 +244,86 @@ def test_sync_target_skips_in_rhiza_repo(self, logger):
+# Check if this is the bump command: "rhiza[tools]>=0.8.6" tools bump
+if "tools" in args and "bump" in args:
+    # Simulate bumping version in pyproject.toml
+    pyproject = Path("pyproject.toml")
+    if pyproject.exists():
+        content = pyproject.read_text()
+        # Simple regex replacement for version
+        # Assuming version = "0.1.0" -> "0.1.1"
+        new_content = re.sub(r'version = "([0-9.]+)"', lambda m: f'version = "{m.group(1)[:-1]}{int(m.group(1)[-1]) + 1}"', content)
+        pyproject.write_text(new_content)
+        print(f"[MOCK] Bumped version in {pyproject}")
+"""  # noqa: E501
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)